core/tauri-utils/src/config.rs

```diff
@@ -1052,6 +1052,9 @@ pub struct WindowConfig {
   /// The window webview URL.
   #[serde(default)]
   pub url: WindowUrl,
+  /// The proxy URL used by the webview for all network requests.
+  #[serde(default = "default_proxy")]
+  pub proxy: Option<String>,
```

Comment:
Let's rename this as well and use `Url` to ensure it is a valid url, also no need for custom default implementation since it is an `Option`
```suggestion
  pub proxy_url: Option<Url>,
```